scripts/server/cromwell.server

```diff
@@ -0,0 +1,122 @@
+#!/bin/sh
```

Comment:
If you haven't already, run these through the shell linter

Reply:
Done.